src/Cake.Common/Tools/DotNetCore/Restore/DotNetCoreRestoreSettings.cs

```diff
@@ -67,6 +67,6 @@ public sealed class DotNetCoreRestoreSettings : DotNetCoreSettings
         /// <summary>
         /// Gets or sets the verbosity of logging to use.
         /// </summary>
-        public DotNetCoreRestoreVerbosity? Verbosity { get; set; }
+        public new DotNetCoreRestoreVerbosity? Verbosity { get; set; }
```

Comment:
This is actually removed in [Restore PR](https://github.com/cake-build/cake/pull/1534/files#diff-b9a11bd3c149fc4a5f99426d52da0e39L143) had to add new so it will build. It is replaced by Verbosity property in DotNetCoreSettings.cs